### ISSUE TITLE
NCS: Add option to fit directly in y-space

### ISSUE THREAD

**Issue by mantid-builder** [mantidproject/mantid#10851]
This issue was originally [TRAC 10009](http://trac.mantidproject.org/mantid/ticket/10009)

Original Reporter: Samuel Jackson

This ticket is blocked by : 
 - [X] [TRAC10032](http://trac.mantidproject.org/mantid/ticket/10032)

Andrew Seel is after an option to fit directly in y-space, skipping the TOF conversion. More discussion with Andrew and Martyn about how this should be done is required.

**Comment by DanNixon** [comment]
Duplicate of #12046
